user/advanced-topics/kde.md

```diff
@@ -18,6 +18,11 @@ still possible to install KDE by issuing this command in dom0:
 ```shell_session
 $ sudo qubes-dom0-update @kde-desktop-qubes
 ```
```

Comment:
Given that 4.0 is EOL, is there any reason to keep the old command that you said doesn't work?